akka-bootstrapper/src/main/resources/reference.conf

```diff
@@ -19,16 +19,16 @@ akka-bootstrapper {
   seed-discovery {
 
     # Class name of the discovery implementation
-    class = "filodb.akkabootstrapper.WhitelistClusterSeedDiscovery"
+    class = "filodb.akkabootstrapper.ExplicitlyListClusterSeedDiscovery"
```

Comment:
`ExplicitlyList` is not grammatically correct.  It should be `ExplicitList` as Explicit is the adjective, and not an adverb.   Another idea is `SpecifiedClusterSeedDiscovery` which is shorter.